<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Shilpi-THINK" name="com.wbqa.wd.test.TestHomePage" tests="8" failures="0" timestamp="7 Jun 2013 19:44:34 GMT" time="31.521" errors="2">
  <testcase name="testBrowseLinks" time="5.309" classname="com.wbqa.wd.test.TestHomePage"/>
  <testcase name="testSRLogo" time="2.919" classname="com.wbqa.wd.test.TestHomePage">
    <error type="java.lang.AssertionError" message="Sony Image not found">
      <![CDATA[java.lang.AssertionError: Sony Image not found
	at org.testng.Assert.fail(Assert.java:94)
	at com.wbqa.wd.utilities.MyAssertion.verifyTrue(MyAssertion.java:16)
	at com.wbqa.wd.test.TestHomePage.testSRLogo(TestHomePage.java:70)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:128)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1203)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1128)
	at org.testng.TestNG.run(TestNG.java:1036)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- testSRLogo -->
  <testcase name="testSearch" time="4.190" classname="com.wbqa.wd.test.TestHomePage"/>
  <testcase name="testSRurl" time="2.667" classname="com.wbqa.wd.test.TestHomePage"/>
  <testcase name="testSRLogo" time="3.586" classname="com.wbqa.wd.test.TestHomePage">
    <error type="java.lang.AssertionError" message="Sony Image not found">
      <![CDATA[java.lang.AssertionError: Sony Image not found
	at org.testng.Assert.fail(Assert.java:94)
	at com.wbqa.wd.utilities.MyAssertion.verifyTrue(MyAssertion.java:16)
	at com.wbqa.wd.test.TestHomePage.testSRLogo(TestHomePage.java:70)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:128)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1203)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1128)
	at org.testng.TestNG.run(TestNG.java:1036)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- testSRLogo -->
  <testcase name="testSRurl" time="3.148" classname="com.wbqa.wd.test.TestHomePage"/>
  <testcase name="testBrowseLinks" time="5.220" classname="com.wbqa.wd.test.TestHomePage"/>
  <testcase name="testSearch" time="4.482" classname="com.wbqa.wd.test.TestHomePage"/>
</testsuite> <!-- com.wbqa.wd.test.TestHomePage -->
